commands/audit/sca/common_test.go

```diff
@@ -36,7 +36,7 @@ func TestGetExcludePattern(t *testing.T) {
 		{
 			name:     "Test no exclude pattern recursive",
 			params:   func() *utils.AuditBasicParams { return (&utils.AuditBasicParams{}).SetIsRecursiveScan(true) },
-			expected: "(^.*\\.git.*$)|(^.*node_modules.*$)|(^.*target.*$)|(^.*venv.*$)|(^.*test.*$)",
+			expected: "(^.*\\.git.*$)|(^.*node_modules.*$)|(^.*target.*$)|(^.*venv.*$)|(^.*test.*$)|(^.*\\.vs.*$)",
```

Comment:
I would not change the default params for the risk of missing info in `.vscode`.
I would instead just pass this param in the VS extension with the added `.vs` so it will only effect VS

(Same for all other changes here to the default params